h2o-docs/src/product/security.rst

```diff
@@ -493,7 +493,7 @@ Example **ldap.conf**:
 ::
 
     ldaploginmodule {
-        org.eclipse.jetty.plus.jaas.spi.LdapLoginModule required
```

Comment:
This break things for the users

Reply:
is there a compatibility layer we could use?

Reply:
I was looking at this today. I wrote custom login module which just extends the new module. That should ensure the users does not need to change their configuration files